aten/src/ATen/native/cpu/Loops.h

```diff
@@ -263,6 +263,7 @@ static inline void vectorized_outer_reduction(char** data, int64_t inner_stride,
 
 template <typename func_t>
 void unary_kernel(TensorIterator& iter, func_t op) {
+  AT_ASSERT(iter.ntensors() > 1);
```

Comment:
naive question: shouldn't `unary` operators have 2 tensors, one for input and one for output?

Reply:
This is check from other PR. Cleaning up.